subsys/can/isotp/Kconfig

```diff
@@ -0,0 +1,129 @@
+# Kconfig - ISO TP configuration options
+
+#
+# Copyright (c) 2019 Alexander Wachter
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig ISOTP
+	bool "ISO-TP"
```

Comment:
What about calling this "ISO-TP Transport for CAN", then user does not really need to see the help text to figure out what this is about.

Reply:
Not sure here. CAN is redundant there because it is in the CANbus subsystem and `ISO-TP Transport for CAN` does not give more information. One who does not know about ISO-TP would still need to look it up. Wikipedia link would be helpful;-)